src/apis/merchandise/ProductsAPI.ts

```diff
@@ -528,14 +528,22 @@ export class ProductsAPI extends BaseAPI {
        return (response as unknown) as ImageOptionsDTO;
    }
 
-   async updateVariant (id: string, dto: ProductVariantUpdateDTO): Promise<ProductVariantDTO> {
+   async setVariantSku (sku: string, dto: ProductSkuUpdateDTO): Promise<void> {
        const response = await this._request({
-           path: `/merchandise/products/variants/${encodeURIComponent(id)}`,
+           path: `/merchandise/products/variants/${encodeURIComponent(sku)}`,
```

Comment:
I've made a change about this endpoint before few days. Looks like you used a not latest platform version. PRs are already merged:
- https://github.com/senteca/platform/pull/851
- https://github.com/senteca/platform/pull/850
- https://github.com/senteca/platform/pull/849

Please update your local platform and regenerate the SDK.

Reply:
@gtsvetanov is it better ?